src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineArtifactStaging.java

```diff
@@ -124,7 +159,23 @@ static void copyAppEngineContext(StageFlexibleConfiguration config, CopyService
     copyService.copyFileAndReplace(appYaml, stagingDirectory.resolve(APP_YAML));
   }
 
-  private static void copyArtifact(StageFlexibleConfiguration config, CopyService copyService)
+  @VisibleForTesting
+  static void copyAppEngineContextStandard(
+      StageArtifactConfiguration config, CopyService copyService)
+      throws IOException, AppEngineException {
+    Path appYaml = config.getAppEngineDirectory().resolve(APP_YAML);
+    if (!Files.exists(appYaml)) {
+      throw new AppEngineException(APP_YAML + " not found in the App Engine directory.");
+    }
+    Path appEngineDirectory = config.getAppEngineDirectory();
+    Path stagingDirectory = config.getStagingDirectory();
+    System.out.println(appEngineDirectory + " " + stagingDirectory);
```

Comment:
should this be a logger? I'm thinking about the usage of this in an IDE environment where the main UI is not System.out.

Reply:
oh this is a mistake, I was testing something.